app/apps/workflow/serializers.py

```diff
@@ -4,20 +4,22 @@
 from drf_spectacular.utils import extend_schema_field
 from rest_framework import serializers
 
-from .models import Task, Workflow
+from .models import CaseUserTask, CaseWorkflow
 
 
-class TaskSerializer(serializers.ModelSerializer):
+class CaseUserTaskSerializer(serializers.ModelSerializer):
     user_has_permission = serializers.SerializerMethodField()
     camunda_task_id = serializers.CharField(source="id")
     form_variables = serializers.Serializer(source="get_form_variables")
+    # frontend dep: rename to 'task_name'
```

Comment:
Not sure what this comment means.

Reply:
frontend dependency. Refactor stops here for now. Next round of refactor involves frontend to.